### PR TITLE
Incomplete reports

### PR DESCRIPTION
**Pull request by elvisciotti**

### CODE REVIEW

File: src/AppBundle/Resources/translations/common.en.yml [hunk 1]
```diff
@@ -26,6 +27,7 @@ manage: Manage
 annualDeputyReport: Annual deputy report
 currentDeputyReport: Current deputy report
 submittedReports: Submitted reports
+unsubmittedReport: Incomplete deputy report
```
Comment (slizzio) [inline src/AppBundle/Resources/translations/common.en.yml]: hmmm key is a bit confusing here

File: src/AppBundle/Service/Client/RestClient.php [hunk 2]
```diff
@@ -280,6 +280,10 @@ public function apiCall($method, $endpoint, $data, $expectedResponseType, $optio
             $options['body'] = $this->toJson($data, $options);
         }
 
+//        if ($method=='put') {
+//            echo "<pre>";\Doctrine\Common\Util\Debug::dump($options, 4);die;
+//        }
+
```
Comment (slizzio) [inline src/AppBundle/Service/Client/RestClient.php]: To remove or remain?

File: src/AppBundle/Resources/translations/common.en.yml (hunk 1 above)
Comment (elvisciotti) [reply src/AppBundle/Resources/translations/common.en.yml]: in the backend, incomplete means a report not submitted, but this new functionality calls them "incomplete" in the UI. I and Biggs decided to keep the naming consistent

File: src/AppBundle/Service/Client/RestClient.php (hunk 2 above)
Comment (elvisciotti) [reply src/AppBundle/Service/Client/RestClient.php]: ups. well spotted. I'll remove this

**Pull request merged**